README.md

```diff
@@ -116,125 +156,56 @@ TensorFlow [prepare environment] for linux.
     ./gtest_ngtf
-    cd python
-    python -m pytest
     ```
-
 Next is to run a few DL models to validate the end-to-end functionality.
 
 2. Go to the ngraph-tf/examples directory and run the following models.
     ```
-    cd examples/resnet
+    cd examples/mnist
     python mnist_fprop_only.py \
-        --data_dir <input_data_location> --select_device NGRAPH
-    python tf_cnn_benchmarks.py --model=resnet50 --eval \
-        --num_inter_threads=1 --batch_size=1 \
-        --train_dir <rep-trained-model-location>/resnet50 \
-        --data_format NCHW --select_device NGRAPH \
-        --num_batches=10
+        --data_dir <input_data_location> 
     ```
 
-<!--
-### Run MNIST Softmax with the activated bridge 
-TODO
--->
-
-
-## OS X Instructions (Experimental)
-
-### Prepare the build environment
-
-1. Install [bazel], as per step 1 in the "Prepare build environment" section 
-above. Note that bazel version [0.11.1 also works]; if you're feeling adventurous, 
-you could try a later version.
-2. Run `$ port install coreutils`, then add `/opt/local/libexec/gnubin` **in front** 
-of your `$PATH`.  Both `tensorflow` and `ngraph` assume GNU userland tools, and 
-you'll run into errors without them.
-3. Make and activate a virtualenv for our custom-built TensorFlow.
-
-
-### Installation
-
-1. Build TensorFlow and its framework for unit tests. This step is identical to 
-how you would build TensorFlow for Linux mentioned above.
-:warning: You must be inside the `virtualenv` created in step 3 above.
+## OS X Instructions
 
-2. Prepare `ngraph-tf` for the build:
-
-	```
-	git clone git@github.com:NervanaSystems/ngraph-tf.git
-	git checkout r0.4
-	pushd ngraph-tf.git
-	ln -s ../tensorflow
-	mkdir build && cd build
-	cmake -DNGRAPH_USE_PREBUILT_LLVM=False ..
-	```
-
-Note: If you want to build a version with no optimization for debugging
-then you can use the `-DCMAKE_BUILD_TYPE=Debug` flag during the cmake step
-mentioned above.
-
-3. `make -j <your-core-count>`
-
-4. `make install`
-
-This step will prepare the pip package for nGraph. Install the `ngraph` package 
-by using the following commands:
-
-5. pip install -U python/dist/ngraph-0.0.0-py2.py3-none-macosx_10_12_x86_64.whl
-
-:warning: The actual file name for the Python whl will vary depending on specific 
-version of macOS.
-
-Verify the installation:
-
-6. python -c "import ngraph"
-
-This step should succeed i.e., you will not see any error message.
+The build and installation instructions are idential for Ubuntu 16.04 and OS X.
 
 ### Running tests
 
-1. Add `<path-to-tensorflow-repo>/bazel-out/darwin-py3-opt/bin/tensorflow` and `<path-to-ngraph-tf-repo>/build/ngraph/ngraph_dist/lib` to your `LD_LIBRARY_PATH` and `DYLD_LIBRARY_PATH`
+1. Add `<path-to-tensorflow-repo>/bazel-out/darwin-py3-opt/bin/tensorflow` and `<path-to-ngraph-tf-repo>/build/ngraph/ngraph_dist/lib` to your `DYLD_LIBRARY_PATH`
 2. Follow the C++ and Python instructions from the Linux based testing described above.
```

Comment:
Follow the instructions from the Linux based testing described above. (We can remove 'C++ and Python' as python tests are not there any more). We have C++ unit tests and python to run models.